src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBulkCSVFileRecord.java

```diff
@@ -183,21 +198,21 @@ public SQLServerBulkCSVFileRecord(String fileToParse, boolean firstLineIsColumnN
         this(fileToParse, null, ",", firstLineIsColumnNames);
     }
 
-    private void initFileReader(InputStreamReader sr, String encoding, String demlimeter,
+    private void initFileReader(InputStreamReader sr, String encoding, String delimeter,
             boolean firstLineIsColumnNames) throws SQLServerException, IOException {
         fileReader = new BufferedReader(sr);
         if (firstLineIsColumnNames) {
             currentLine = fileReader.readLine();
             if (null != currentLine) {
-                columnNames = currentLine.split(delimiter, -1);
+                columnNames = delimiterPattern.split(currentLine, -1);
```

Comment:
Please explain this change.

Reply:
using a compiled pattern thinking it would be faster?